r1cs/gadgets/std/src/groups/mod.rs

```diff
@@ -598,6 +598,23 @@ pub(crate) mod test {
             .unwrap();
         assert!(cs.is_satisfied());
 
+        // testing consistency with primitive arithmetic
```

Comment:
Can we do this changing the value of a and b, starting from a rng, and for a certain number of iterations ?

Reply:
Maybe also for the mul_bits

Reply:
Ok